ClientLib/src/main/java/com/o3dr/services/android/lib/drone/mission/item/complex/SurveyDetail.java

```diff
@@ -56,6 +57,14 @@ public void setCameraDetail(CameraDetail cameraDetail) {
         this.cameraDetail = cameraDetail;
     }
 
+    public boolean getLockOrientation(){
```

Comment:
@TShapinsky can you add javadocs to both methods describing their use, and specifying when (which client lib version) they were added in (should be version 3.0 in this case)